ui_tests/pages/profiles_page.py

```diff
@@ -0,0 +1,299 @@
+import time
+
+import pages.helper as helper
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class ProfilesPage:
+    def checkIfProfilesTableIsDisplayed(self):
```

Comment:
can we change the name to more pythonic syntax?

Reply:
done